pkg/apis/monitoring/v1/types.go

```diff
@@ -401,6 +401,12 @@ type PrometheusSpec struct {
 	// entire scrape will be treated as failed. 0 means no limit.
 	// Only valid in Prometheus versions 2.27.0 and newer.
 	EnforcedLabelValueLengthLimit *uint64 `json:"enforcedLabelValueLengthLimit,omitempty"`
+	// Minimum number of seconds for which a newly created pod should be ready
+	// without any of its container crashing for it to be considered available.
+	// Defaults to 0 (pod will be considered available as soon as it is ready)
+	// This is an alpha field and requires enabling StatefulSetMinReadySeconds feature gate.
+	// +optional
+	MinReadySeconds *uint32 `json:"minReadySeconds,omitempty"`
```

Comment:
Could it be `uint32`? Given that the default value is 0, I don't think we need a pointer here.

Reply:
It could. Happy to change it. Was just following the couple of fields above and thought being explicit about being optional might be the way but I'm good with changing it.

Reply:
You're talking about the `enforced*` fields? I think that nobody really pay attention initially and we've kept copying the same pattern... I assume that we could switch to non-pointer values without user-facing impact.

Reply:
yep, done